doc/js-api/remotestorage.rst

```diff
@@ -70,19 +70,19 @@ List of events
 
 ``wire-busy``
 """""""""""""
-   Emitted when a wire request starts
+   Emitted when a network request starts
 
 ``wire-done``
 """""""""""""
-   Emitted when a wire request completes
+   Emitted when a network request completes
 
 ``sync-req-done``
 """""""""""""""""
-   Emitted when a single sync task has finished
+   Emitted when a sync request has finished
 
 ``sync-done``
 """""""""""""
-   Emitted when the complete sync cycle has finished
+   Emitted when a sync cycle has finished
```

Comment:
Again, not sure if removing "complete" is useful here, making it more ambiguous. I'd probably change "finished" to "completed" even, so it's clear when this is emitted. In a comment I found earlier, it also says "... and a new sync is scheduled", making it even clearer.

Reply:
I like that description, will change it